### PR TITLE
Redirect user based on current path after logging in

### PR DESCRIPTION
**Pull request by cameronmarsh**
smartcolumbusos/discovery_ui#20

co-authored-by: Scott Millard <smillard@hntb.com>

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -7661,7 +7661,7 @@
     },
     "external-editor": {
       "version": "2.2.0",
-      "resolved": "https://registry.npmjs.org/external-editor/-/external-editor-2.2.0.tgz",
+      "resolved": "http://registry.npmjs.org/external-editor/-/external-editor-2.2.0.tgz",
```
Comment (JarredOlson) [inline package-lock.json]: This is a setting that has bitten us previously.  These should all be `https` not `http`

Comment (jmorris-pillar) [inline package-lock.json]: make sure these do not get rolled to http

Comment (cameronmarsh) [reply package-lock.json]: yeah..

Comment (cameronmarsh) [reply package-lock.json]: gotcha

**Pull request merged**